plugins/sidebar/overlays/new-file.js

```diff
@@ -1,47 +1,24 @@
 'use strict';
 
+const _ = require('lodash');
 const React = require('react');
 const Card = require('react-material/components/Card');
 const Button = require('react-material/components/Button');
 const TextField = require('react-material/components/TextField');
 
+const FileActions = require('../../../src/actions/FileActions.js');
```

Comment:
stores and actions are instances of classes, convention says they should start with a lowercase letter